scipy/optimize/_basinhopping.py

```diff
@@ -248,12 +249,22 @@ class RandomDisplacement(object):
     Add a random displacement of maximum size, stepsize, to the coordinates
 
     update x inplace
+
+    Parameters
+    ----------
+    stepsize : float, optional
+        stepsize
+    rand_state : None or `np.random.RandomState` instance, optional
+        The random number generator that generates the displacements
     """
-    def __init__(self, stepsize=0.5):
+    def __init__(self, stepsize=0.5, rand_state=None):
```

Comment:
Bikeshedding: scikit-learn names this argument `random_state`. scipy.stats distributions follow. Maybe it's better to be consistent here.